bin/rm.sh

```diff
@@ -208,51 +208,51 @@ fi
 remove(){
   local file=$1
 
-  # if is dir
-  if [[ -d "$file" ]]; then
-
-  # if a directory, and without '-r' option
-  if [[ ! -n "$OPT_RECURSIVE" ]]; then
-    debug "$LINENO: $file: is a directory"
-    echo "$COMMAND: $file: is a directory"
-    return 1
-  fi
+  # if is dir and not a symbolick link
+  if [[ -d "$file" && ! -L "$file" ]]; then
+
+    # if a directory, and without '-r' option
```

Comment:
Added one level of indentation: those are clauses evaluated in `directory` condition